app/src/pages/playground/PlaygroundTemplate.tsx

```diff
@@ -58,10 +53,7 @@ export function PlaygroundTemplate(props: PlaygroundTemplateProps) {
             <PromptComboBox
               promptId={promptId}
               onChange={(nextPromptId) => {
-                updateInstancePrompt({
-                  instanceId,
-                  patch: nextPromptId ? { id: nextPromptId } : null,
-                });
+                navigate(`/prompts/${nextPromptId}/playground`);
```

Comment:
the problem with this approach is that it means you can't load two different prompts in the UI at once. I would leave the navigation static here.

Reply:
Fixed